psi4/src/psi4/libmints/transform.cc

```diff
@@ -80,12 +80,13 @@ void SphericalTransform::init() {
 
     // Go through and grab the values.
     int pureindex = 0;
-    int cartindex = 0;
+    //int cartindex = 0; this is not used
```

Comment:
Nice. Go ahead and delete the lines not used. Thank you of the documentation you added, too!

Reply:
Can I also delete commented out code? It is confusing.

Reply:
sure, thanks